{"name": "Проверка получения ошибки при дублировании одного логина в создании курьера", "status": "passed", "start": 1730055277468, "stop": 1730055277579, "uuid": "a8f2a36b-c35e-4d40-a13b-3542887d4c32", "historyId": "b2efc793811c8c331ab464f3f831c46b", "testCaseId": "b2efc793811c8c331ab464f3f831c46b", "fullName": "tests.test_create_courier.Test_create_courier#test_create_courier_account_login_taken_conflict", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "Test_create_courier"}, {"name": "host", "value": "MacBook-Air-Natasha.local"}, {"name": "thread", "value": "80125-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}